src/OrchardCore.Build/Dependencies.props

```diff
@@ -27,6 +27,7 @@
     <PackageManagement Include="Newtonsoft.Json" Version="12.0.2" />
     <PackageManagement Include="NLog.Web.AspNetCore" Version="4.8.5" />
     <PackageManagement Include="NodaTime" Version="2.4.7" />
+    <PackageManagement Include="OrchardCore.Translations.All" Version="1.0.0-beta3.100037" />
```

Comment:
I don't know if we need a `*` resolution here, like `1.0.0-beta3.*`

Also did we make a mistake by creating version like `.100037` instead of `-100037` ?

Reply:
Oh, I also need to push the packages on https://www.myget.org/feed/Packages/orchardcore-dev as this is the feed that OC uses for its "source" packages. They also need to be on the preview feed for the end users. That's why it can't find it.

Reply:
> Also did we make a mistake by creating version like .100037 instead of -100037 ?

If I recall you was so quick at last point when you change the `pack.ps1` :) anyhow it is fixed in https://github.com/OrchardCMS/OrchardCore.Translations/pull/26